nix/ext/citus.nix

```diff
@@ -0,0 +1,37 @@
+{ lib, curl, lz4, zstd, krb5, icu, stdenv, fetchFromGitHub, postgresql }:
```

Comment:
do we still need this one?

Reply:
@olirice awesome,  thanks for reviewing this! I put this together while I was stuck for some hours in CDG Paris Airport :-D

I'll just address your feedback in some commits and get the actions integrated here ASAP, and then we could merge it with all that feedback plus actions migrated over. So I will just start resolving this all now.

Reply:
This one is resolved with https://github.com/supabase/postgres/pull/909/commits/73bee62e6ec161055ba75056e9c9a4585f2cadc9